capstone/src/main/java/com/google/univiz/collegescorecarddata/MapData.java

```diff
@@ -0,0 +1,8 @@
+package com.google.univiz.collegescorecarddata;
+
+import java.util.List;
+
+public interface MapData {
+  /** Maps Feature Functions */
+  List<UniversityData> getMapData(List<UniversityData> collegeDatum);
```

Comment:
what does it mean for a `List<UniversityData>` to produce a `List<UniversityData>`?

Reply:
made some comments on the new file!